[FIX] tighter rules for sharing MEG cross-talk and fine-calibration .fif files (#598)
bids-standard/bids-specification: src/99-appendices/06-meg-file-formats.md

In BIDS, the cross-talk and fine-calibration files are shared unmodified,
but with BIDS file naming convention and by using the `acq` entity.

-   cross-talk file template: `sub-<label>[_ses-<label>]_acq-crosstalk_meg.dat`
-   fine-calibration file template: `sub-<label>[_ses-<label>]_acq-calibration_meg.fif`

Note that cross-talk files MUST be denoted using `acq-crosstalk` and
fine-calibration files MUST be denoted using `acq-calibration`.

> The cross-talk and fine-calibration data MUST be stored in the subject level `meg` folder,
> optionally nested in a subject specific `ses-<label>` folder, as shown in the following examples.

Reviewer: ```suggestion
The cross-talk and fine-calibration data MUST be stored in the subject-level `meg` folder,
which may be nested inside a `ses-<label>` folder, as shown in the following examples.
```

            sub-control01_ses-01_coordsystem.json
            sub-control01_ses-01_task-rest_run-01_meg.fif
            sub-control01_ses-01_task-rest_run-01_meg.json
            sub-control01_ses-01_task-rest_run-01_channels.tsv
            sub-control01_ses-01_acq-crosstalk_meg.dat
            sub-control01_ses-01_acq-calibration_meg.fif
    ses-02/
        sub-control01_ses-02_scans.tsv
        meg/
            sub-control01_ses-02_coordsystem.json
            sub-control01_ses-02_task-rest_run-01_meg.fif
            sub-control01_ses-02_task-rest_run-01_meg.json
            sub-control01_ses-02_task-rest_run-01_channels.tsv
            sub-control01_ses-02_acq-crosstalk_meg.dat
            sub-control01_ses-02_acq-calibration_meg.fif
```

#### Example 2

```Text
sub-01/
    meg/
        sub-01_coordsystem.json
        sub-01_task-rest_meg.fif
        sub-01_task-rest_meg.json
        sub-01_task-rest_channels.tsv
        sub-01_acq-crosstalk_meg.dat
        sub-01_acq-calibration_meg.fif
sub-02/
    meg/
        sub-02_coordsystem.json
        sub-02_task-rest_meg.fif
        sub-02_task-rest_meg.json
        sub-02_task-rest_channels.tsv
        sub-02_acq-crosstalk_meg.dat

>         sub-02_acq-calibration_meg.fif

Reviewer: Can you swap the order of Examples 1 and 2, so we start with the simpler nesting structure?

Reviewer: Also I think I would equalize the subjet names between both examples so it's easier for a reader to spot where the meaningful differences are

Author: yes, good point - will do tomorrow!
